README.md

```diff
@@ -180,8 +180,8 @@ More details are available [here](https://github.com/ceph/ceph-csi/issues/463)
 
 ## Dev standup
 
-A regular dev standup takes place every [Monday,Tuesday and Thursday at
-12:00 PM UTC](https://meet.google.com/nnn-txfp-cge). Convert to your local
+A regular upstream dev standup takes place every [Tuesday at
```

Comment:
this is a community project, there is no need to make a difference between upstream/downstream as it is per definition about upstream :smiley:

Reply:
Thanks for the review Niels, updated.